Nos/Views/Discover/FeaturedAuthor.swift

```diff
@@ -1,3 +1,4 @@
+// swiftlint:disable file_length
```

Comment:
Could we consider breaking this into multiple files now that it's gotten so large?

Reply:
I'm not sure what to name that file. `FeaturedAuthor+Extension`?

Reply:
I'm not 100% sure, but I might do one per group/cohort, then name it like `FeaturedAuthor+Cohort1`. Or if there aren't too many total authors, it might make sense to  move all the arrays into one file? Like `FeaturedAuthor+Cohorts`?

Reply:
I love the `FeaturedAuthor+Cohort1` idea. Thank you!